command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
```

Comment:
Use fmt.Println instead of fmt.Printf here because there are no format specifiers in the string.